### PR TITLE
Feature/deploy commit

### PR DESCRIPTION
**Pull request by PaulBrachmann**

### CODE REVIEW

File: packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st [hunk 1]
```diff
@@ -0,0 +1,13 @@
+request handling
+handleUpdate: aWebRequest
+	self authenticate: aWebRequest action: [
```
Comment (arminWel) [inline packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st:3]: This method is very similar to the handleDeploy`. Is it possible to merge those methods?

**Review by arminWel — COMMENTED** [review]
Well implemented feature. I'd like to see some tests for that, if possible. Otherwise you got my go ahead.

File: packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st (hunk 1 above)
Comment (PaulBrachmann) [reply packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st:3]: I've tried to extract common logic into `parseURL:with:`.

**Comment by PaulBrachmann** [comment]
> Well implemented feature. I'd like to see some tests for that, if possible. Otherwise you got my go ahead.

As mentioned before, the deploy agent is not expected to change very frequently and additionally is quite hard to test automatically. As of now, it is thus tested manually.

Integration into the test suite may, however, be a great idea for the future.

File: packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st (hunk 1 above)
Comment (arminWel) [reply packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st:3]: Looks logic. But I don't understand why you need `tokens` as the parameter if the blocks. They are not used in the first block at least

Comment (PaulBrachmann) [reply packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st:3]: This way, the tokens don't have to be parsed multiple times.

Removing the parameter for one of the blocks would yield an error message along the lines of this: `Error: This block accepts 0 arguments, but was called with 1 argument.`

**Pull request merged**